developer-docs-site/docs/tutorials/building-wallet-extension.md

```diff
@@ -34,19 +34,31 @@ The wallet has implemented some of the basics of interacting with Aptos
 - Fund your account with test coins
 - Send coins to another address
 - Link to your account resources on Explorer
+- View and Mint NFTs
```

Comment:
View and Create. When i think of Mint i assume there's a minting and "claiming" process involved, so far the functionality is just for depositing an NFT into the owner's account